src/function.jl

```diff
@@ -5,7 +5,10 @@ struct AutoForwardDiff <: AbstractADType end
 struct AutoReverseDiff <: AbstractADType end
 struct AutoTracker <: AbstractADType end
 struct AutoZygote <: AbstractADType end
-struct AutoFiniteDiff <: AbstractADType end
+struct AutoFiniteDiff <: AbstractADType 
+    difftype::Symbol
+end
+AutoFiniteDiff() = AutoFiniteDiff(:central)
```

Comment:
default to forward

Reply:
Okay 👍

Reply:
What about the type for the hessian?

Reply:
it can only be calculated with central, so you can just assume that. Note that you're going to want to store the information as its done in FiniteDiff since otherwise this is not type-stable.